lib/logger.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+// Copyright (C) 2020 Splunk, Inc. All rights reserved.
+
+var winston = require('winston');
+
+var logger = winston.createLogger({
+  level: 'info',
```

Comment:
Does it make it a fixed config? Can user override log level using ENV vars or some other mechanism?

Reply:
Added SFX_CLIENT_LOG_LEVEL.